src/State.ts

```diff
@@ -27,7 +33,7 @@ export const useState = createGlobalState(() =>
       tasks: Array<Task>(),
       dailyReset: "12:00",
     },
-    version: "0.2.0",
+    version: "0.2.4",
```

Comment:
We don't want to change this value because this makes sure that we always migrate data from before we had this method for state. If you want for clarity, you can change this to a constant.